app/scripts/modules/core/src/pipeline/config/triggers/pipeline/PipelineTrigger.tsx

```diff
@@ -1,64 +1,60 @@
 import * as React from 'react';
 import Select, { Option } from 'react-select';
 
-import { Observable } from 'rxjs';
+import { Observable, Subject } from 'rxjs';
 
-import { ApplicationReader } from 'core/application/service/ApplicationReader';
-import { BaseTrigger } from 'core/pipeline';
+import { Application, ApplicationReader } from 'core/application';
+import { BaseTrigger, PipelineConfigService } from 'core/pipeline';
 import { IPipeline, IPipelineTrigger } from 'core/domain';
-import { PipelineConfigService } from 'core/pipeline/config/services/PipelineConfigService';
+import { Omit } from 'core/presentation';
 import { Checklist } from 'core/forms';
-import { Application } from '@spinnaker/core';
-import { Subject } from 'rxjs/Subject';
+
+type IPipelineTriggerConfig = Omit<IPipelineTrigger, 'parentExecution' | 'parentPipelineId' | 'user' | 'rebake'>;
```

Comment:
this was a maybe-temporary way of accurately typing what's really an acceptable input to `onTriggerUpdated`. I'd like to investigate how feasible having config-only trigger interfaces would be so we can use those in configs rather than the interfaces for triggers that show up in executions.